### ISSUE TITLE
Text for MIDIOutput.send() implies that passing a Uint8Array will be efficient, but that's not the case

### ISSUE THREAD

**Issue by bzbarsky** [WebAudio/web-midi-api#142]
The prose for `MIDIOutput.send()` says:

```
while still enabling use of Uint8Arrays for efficiency in large MIDI data scenarios
```

but it doesn't allow that.  Actually using a Uint8Array efficiently for a sequence argument is pretty difficult, and I'm not aware of any UA that implements the spec for sequences correctly and has such efficient handling.

If you want to allow efficient passing of Uint8Array here, you probably want to define an overload to that effect, or a union with it or something.  Note that this will have observably different behavior from just using sequence for Uint8Array in some edge cases that you probably don't care about (e.g. when someone overrides the array iterator's next() method).

**Comment by domenic** [comment]
I may be opening a can of worms here but you may want to detach the buffer if a Uint8Array is indeed given, so that implementations can avoid a copy.

**Comment by cwilso** [comment]
I can 
1) remove the comment about efficiency
2) add an overload, FOR efficiency.  optionally detach the buffer.

Thoughts?

**Comment by cwilso** [comment]
Remove comment.

**Comment by mjwilson-google** [comment]
This was fixed with pull request #237
